docs/conf.py

```diff
@@ -14,18 +14,46 @@
 
 from datetime import datetime
 import os
+from sphinx.directives.code import CodeBlock
 import sphinx_rtd_theme
 import sys
 
 
+# https://stackoverflow.com/questions/44761197/how-to-use-substitution-definitions-with-code-blocks
+class SubstitutionCodeBlock(CodeBlock):  # type: ignore
+  """
+  Similar to CodeBlock but replaces placeholders with variables.
+  """
+
+  def run(self):
+    """
+    Replace placeholders with given variables.
```

Comment:
Environment variables?

Reply:
I don't think they are really environment variables? I can make the comment a bit better.